src/dashboard/src/components/archival_storage/views.py

```diff
@@ -613,8 +574,78 @@ def document_json_response(document_id_modified, type):
     pretty_json = json.dumps(json.loads(data), sort_keys=True, indent=2)
     return HttpResponse(pretty_json, content_type='application/json')
 
+
 def file_json(request, document_id_modified):
     return document_json_response(document_id_modified, 'aipfile')
 
+
 def aip_json(request, document_id_modified):
     return document_json_response(document_id_modified, 'aip')
+
+
+def view_aip(request, uuid):
+    es_client = elasticSearchFunctions.get_client()
+    try:
+        es_aip_doc = elasticSearchFunctions.get_aip_data(es_client, uuid, fields='name,size,created,status,filePath')
+    except IndexError:
+        raise Http404
+
+    name = _get_es_field(es_aip_doc['fields'], 'name')
+    active_tab = None
+
+    form_upload = forms.UploadMetadataOnlyAtomForm(prefix='upload')
+    form_reingest = forms.ReingestAIPForm(prefix='reingest')
+    form_delete = forms.DeleteAIPForm(prefix='delete', uuid=uuid)
+
+    # Process metadata-only DIP upload form
+    if request.POST and 'submit-upload-form' in request.POST:
+        form_upload = forms.UploadMetadataOnlyAtomForm(request.POST, prefix='upload')
+        active_tab = 'upload'
+        if form_upload.is_valid():
+            try:
+                file_slug = upload_dip_metadata_to_atom(name, uuid, form_upload.cleaned_data['slug'])
+            except AtomMetadataUploadError:
+                messages.error(request, 'Metadata-only DIP upload failed, check the logs for more details')
+            else:
+                messages.success(request, 'Metadata-only DIP upload has been completed successfully. New resource has slug: {}'.format(file_slug))
+            form_upload = forms.UploadMetadataOnlyAtomForm(prefix='upload')  # Reset form
+
+    # Process reingest form
+    if request.POST and 'submit-reingest-form' in request.POST:
+        form_reingest = forms.ReingestAIPForm(request.POST, prefix='reingest')
+        active_tab = 'reingest'
+        if form_reingest.is_valid():
+            response = storage_service.request_reingest(uuid, form_reingest.cleaned_data['reingest_type'])
+            error = response.get('error', True)
+            message = response.get('message', 'An unknown error occurred.')
+            if error:
+                messages.error(request, 'Error re-ingesting package: {}'.format(message))
+            else:
+                messages.success(request, message)
+            return redirect('archival_storage_index')
+
+    # Process delete form
+    if request.POST and 'submit-delete-form' in request.POST:
+        form_delete = forms.DeleteAIPForm(request.POST, prefix='delete', uuid=uuid)
+        active_tab = 'delete'
+        if form_delete.is_valid():
+            response = storage_service.request_file_deletion(uuid, request.user.id, request.user.email, form_delete.cleaned_data['reason'])
+            messages.info(request, response['message'])
```

Comment:
Elasticsearch deletion seems to have gotten lost.  Original aip_delete included `es_client = elasticSearchFunctions.get_client(); elasticSearchFunctions.mark_aip_deletion_requested(es_client, uuid)` (see above)